server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheContainerConfigurationBuilder.java

```diff
@@ -116,6 +120,18 @@ public ServiceName getServiceName() {
                 .addDependency(ScheduledThreadPoolResource.EXPIRATION.getServiceName(this.name), ThreadPoolConfiguration.class, this.expirationThreadPool)
                 .addDependency(ScheduledThreadPoolResource.REPLICATION_QUEUE.getServiceName(this.name), ThreadPoolConfiguration.class, this.replicationQueueThreadPool)
         ;
+        if (module != null) {
+            if (!module.getName().equals("org.infinispan.extension")) {
+                // todo [anistor] only works for dynamic modules
```

Comment:
I noticed these similar todos around. Is this something we need to do in this PR or do we need a different JIRA?

Reply:
Yes, this should be a different JIRA, not reported yet. Will report it and update the comments with the id.